### ISSUE TITLE
The Quick Reference role filter is ignored when navigating from role pages

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wai-website#581]
### Steps to Reproduce

Seen in Chrome 76.

1. Start on one of these role pages: [Content Writers](https://www.w3.org/WAI/roles/writers/), [Designers](https://www.w3.org/WAI/roles/designers/), or [Developers](https://www.w3.org/WAI/roles/developers/)
2. Locate a filter link, such as 'How to Meet WCAG 2 (Quick Reference) – filtered for “Visual Design” Tags'. Note the link URL:
`https://www.w3.org/WAI/WCAG20/quickref/?currentsidebar=%23col_customize&tags=animation%2Cblinking%2Ccolor%2Ccontrast%2Cflashing%2Chidden-content%2Cimages%2Cimages-of-text%2Clayout%2Cprogress-steps%2Cstructure%2Ctables%2Ctext%2Ctext-alternatives%2Cvideo%2Czoomh`
3. Activate the link.
4. Activate the "Filter" tab and observe the content under Tags.

### Actual Behavior

At Step 3:

In each case the browser is redirected to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0

A bar near the top says "Selected Filters: WCAG 2.0: all success criteria and all techniques."

The Quick Reference page is not filtered by tags for the role.

At Step 4:

The checkboxes for all roles are checked.

No tags are selected.

### Expected Behavior

At Step 3, the Quick Reference page should appear filtered by tags for the requested role.

At Step 4, only the checkbox for the requested role should be checked, and tags for that one role should be selected.
